cmd/ib-kubernetes/main.go

```diff
@@ -4,20 +4,38 @@ import (
 	"flag"
 	"os"
 
-	"github.com/golang/glog"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
 
 	"github.com/Mellanox/ib-kubernetes/pkg/daemon"
 )
 
+func setupLogging(debug bool) {
+	if debug {
+		zerolog.SetGlobalLevel(zerolog.DebugLevel)
+	} else {
+		zerolog.SetGlobalLevel(zerolog.InfoLevel)
+	}
+	log.Logger = log.Output(zerolog.ConsoleWriter{
+		Out:        os.Stderr,
+		TimeFormat: zerolog.TimeFieldFormat,
+		NoColor:    true})
+}
+
 func main() {
+	debug := false
```

Comment:
you can just : `var debug bool`
its default is already false and you are initializing it with false in `flag.BoolVar()` below

Reply:
fixed